<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html><head>
<link rel="canonical" href="https://www.ncbi.nlm.nih.gov/Traces/sra?run=ERR004400">
<title>Run Browser : Browse  :
            Sequence Read Archive  :
            NCBI/NLM/NIH</title>
<link href="https://www.ncbi.nlm.nih.gov/favicon.ico" rel="shortcut icon" type="image/x-icon">
<meta name="keywords" content="sra, sequence read archive, national center for biotechnology information, ncbi,          national library of medicine, nlm, national institutes of health, nih, database, archive, bioinformatics, biomedicine,          trace archive, whole genome shotgun, raw sequence, megablast">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="robots" content="index,nofollow,noarchive">
<meta http-equiv="X-UA-Compatible" content="edge,chrome=1">
<meta name="galert_type" content="default">
<script type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/jquery/jquery-1.12.4.min.js"></script><script type="text/javascript">
            jQuery.getScript("/core/alerts/alerts.js", function () {
            galert(['div#gsd'])
            });
        </script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/utils.1.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/stop_js_include.1.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/firebugx.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/notify.1.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/data_provider.1.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/remote_data_provider.1.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/gui.3.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/session.1.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/tabmenu.1.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/panel.1.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/window.1.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/toggler.1.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/expander.1.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/sort_table.1.js"></script><script type="text/javascript" src="../cluster/web/../../core/lw/2.0/js/mail_form.1/mail_form.1.js"></script><script type="text/javascript" src="../cluster/web/cluster.js"></script><link href="../cluster/web/../../core/lw/2.0/css/gui.3/gui.3.css" type="text/css" rel="stylesheet">
<link href="../cluster/web/../../core/lw/2.0/css/gui.3/expander.1.css" type="text/css" rel="stylesheet">
<link href="../cluster/web/../../core/lw/2.0/css/gui.3/tabmenu.1.css" type="text/css" rel="stylesheet">
<link href="../cluster/web/../../core/lw/2.0/css/mail_form.1/mail_form_trace.css" type="text/css" rel="stylesheet">
<link href="../cluster/web/global.css" type="text/css" media="all" rel="stylesheet">
<link href="../cluster/web/cluster.css" type="text/css" media="all" rel="stylesheet">
<title>Sequence Read Archive</title>
<link href="style.css" type="text/css" media="all" rel="stylesheet">
<script type="text/javascript" src="../core/lw/2.0/js/web_forms.1.js"></script><script type="text/javascript" src="../core/lw/2.0/js/esummary.1.js"></script><script type="text/javascript" src="js/esummary_sra.js"></script><script type="text/javascript" src="js/sra.js"></script><link href="js/viewer.css" type="text/css" media="all" rel="stylesheet">
<link href="../core/lw/2.0//css/gui.3/tab.1.css" type="text/css" media="all" rel="stylesheet">
<script type="text/javascript" src="../core/lw/2.0/js/scroll_table.1.js"></script><script type="text/javascript" src="../core/lw/2.0/js/gui.3.js"></script><script type="text/javascript" src="../core/lw/2.0/js/session.1.js"></script><script type="text/javascript" src="../core/lw/2.0/js/tab.1.js"></script><script type="text/javascript" src="../core/lw/2.0/js/tree_static.1.js"></script><script type="text/javascript" src="js/spot_signal.js"></script><script type="text/javascript" src="js/spinner.js"></script><script type="text/javascript" src="js/pager.js"></script><script type="text/javascript" src="js/spot_list.js"></script><script type="text/javascript" src="js/spot_data.js"></script><script type="text/javascript" src="js/spot_customize.js"></script><script type="text/javascript" src="js/app.js"></script>
</head><body><a href="#maincontent" id="skip-to-maincontent">skip to main content</a><div id="header-global">
<div id="site-logo" class="branding"><strong id="ncbi-logo" class="logo"><a href="https://ncbi.nlm.nih.gov" accesskey="1" title="NCBI Home (access key '1')"><img src="../cluster/web//i/ncbi-logo-small_369_.gif" alt="NCBI" align="absmiddle"></a></strong></div>
<div id="global-nav"><form method="get" action="https://www.ncbi.nlm.nih.gov/entrez" target="_blank"><ul class="pipelist2l">
<li><a href="https://www.ncbi.nlm.nih.gov/Sitemap/index.html" title="NCBI Site Map">Site map</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/gquery/gquery.fcgi" title="Global Cross-database NCBI Search">All databases</a></li>
<li><a href="https://www.ncbi.nlm.nih.gov/feed/rss.cgi?ChanKey=sranews" title="Brief annoucements for new features and pipeline changes to the Sequence Read Archive."><img src="../cluster/web/i/feed-icon-14x14.png" alt="RSS Feed" align="absmiddle" border="0"></a></li>
<li id="ph-global-search">
<a href="#gloabal-search" onclick="return Cluster.toggleSearch(this)">Search</a> <span id="gloabal-search" class="invis"><label for="term">
                                for <input type="text" name="term" id="term" size="24"></label> <label for="id-db">
                                within <select name="db" id="id-db"><option value="pubmed">PubMed</option>
<option value="protein">Protein</option>
<option value="nuccore">Nucleotide</option>
<option value="nucgss">GSS</option>
<option value="nucest">EST</option>
<option value="structure">Structure</option>
<option value="genome">Genome</option>
<option value="assembly">Assembly</option>
<option value="bioproject">BioProject</option>
<option value="biosample">BioSample</option>
<option value="books">Books</option>
<option value="cdd">Conserved Domains</option>
<option value="clinvar">ClinVar</option>
<option value="clone">Clone</option>
<option value="gap">dbGaP</option>
<option value="dbvar">DbVar</option>
<option value="epigenomics">Epigenomics</option>
<option value="gene">Gene</option>
<option value="gds">GEO Datasets</option>
<option value="geoprofiles">GEO Profiles</option>
<option value="homologene">HomoloGene</option>
<option value="medgen">MedGen</option>
<option value="mesh">MeSH</option>
<option value="nlmcatalog">NLM Catalog</option>
<option value="ncbisearch">NCBI Web Site</option>
<option value="omia">OMIA</option>
<option value="omim">OMIM</option>
<option value="pmc">PMC</option>
<option value="popset">PopSet</option>
<option value="probe">Probe</option>
<option value="proteinclusters">Protein Clusters</option>
<option value="pcassay">PubChem BioAssay</option>
<option value="biosystems">BioSystems</option>
<option value="pccompound">PubChem Compound</option>
<option value="pcsubstance">PubChem Substance</option>
<option value="pubmedhealth">PubMed Health</option>
<option value="snp">SNP</option>
<option value="sra">SRA</option>
<option value="taxonomy">Taxonomy</option>
<option value="toolkit">ToolKit</option>
<option value="toolkitall">ToolKitAll</option>
<option value="toolkitbook">ToolKitBook</option>
<option value="unigene">UniGene</option>
<option value="unists">UniSTS</option></select></label> <input type="submit" name="submit" value="Search"></span>
</li>
</ul></form></div>
</div><div id="header-local">
<div id="prj-logo" class="branding"><strong id="sra-logo" class="logo"><a title="to Sequence Read Archive Main Page" href="../sra"><span> </span><img src="i/sra_logo_eee.gif" alt="Sequence Read Archives logo"></a></strong></div>
<div id="toolbox"></div>
<div id="local-nav"><ul class="tabmenu" id="tabmenu">
<li>
<a title="Sequence Read Archive Main Page" href="../sra/sra.cgi?view=announcement"><span>Main</span></a><ul>
<li><a title="Announcements, Notifications and News" href="../sra/sra.cgi?view=announcement"><span>Overview</span></a></li>
<!--[if gte IE 5]><iframe></iframe><![endif]-->
</ul>
</li>
<li id="tabmenu_current" class="selected">
<a title="Get SRA Data" href="../sra/sra.cgi?view=studies"><span>Browse</span></a><ul>
<li><a title="List of Sequence Read Archive Studies" href="../sra/sra.cgi?view=studies"><span>Studies</span></a></li>
<li><a title="List of Sequence Read Archive Samples" href="../sra/sra.cgi?view=samples"><span>Samples</span></a></li>
<li><a title="List of Sequence Read Archive Analyses" href="../sra/sra.cgi?view=analysis"><span>Analyses</span></a></li>
<li class="subcurrent"><a title="Sequence Read Archive Run Browser" href="../sra/sra.cgi?view=run_browser"><span>Run Browser</span></a></li>
<li><a title="Select Runs using Run Selector Tool" href="../study/?go=home"><span>Run Selector</span></a></li>
<li><a title="Submissions to the Provisional Sequence Read Archive" href="../sra/sra.cgi?view=provisional"><span>Provisional SRA</span></a></li>
<!--[if gte IE 5]><iframe></iframe><![endif]-->
</ul>
</li>
<li>
<a title="Search and Filter individual reads" href="../sra/sra.cgi?view=search_obj"><span>Search</span></a><ul>
<li><a title="Search for Short Read Objects" href="../sra/sra.cgi?view=search_obj"><span>SRA Objects</span></a></li>
<li><a title="Search SRA sequences using BLAST" href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PROGRAM=blastn&amp;BLAST_PROGRAMS=megaBlast&amp;PAGE_TYPE=BlastSearch&amp;BLAST_SPEC=SRA&amp;SHOW_DEFAULTS=on&amp;DB_GROUP=Exp&amp;view=search_blast"><span>BLAST</span></a></li>
<li><a title="Search NCBI's Entrez system of interlinked databases in various ways" href="../sra/sra.cgi?view=search_entrez"><span>Entrez</span></a></li>
<!--[if gte IE 5]><iframe></iframe><![endif]-->
</ul>
</li>
<li>
<a title="Download Data" href="../sra/sra.cgi?view=download_reads"><span>Download</span></a><ul>
<li><a title="Downloading SRA data in either fasta or fastq format" href="../sra/sra.cgi?view=search_seq_name"><span>FASTA/FASTQ</span></a></li>
<li><a title="Download Reads in SRA Formats" href="../sra/sra.cgi?view=download_reads"><span>Reads</span></a></li>
<li><a title="Download data by Analysis" href="../sra/sra.cgi?view=download_analyses"><span>Analyses </span></a></li>
<li><a title="Mirroring SRA Data" href="../sra/sra.cgi?view=mirroring"><span>Mirroring</span></a></li>
<li><a title="References Collection" href="../sra/sra.cgi?view=references"><span>References</span></a></li>
<!--[if gte IE 5]><iframe></iframe><![endif]-->
</ul>
</li>
<li>
<a title="" href="../sra_sub/sub.cgi?view=submissions"><span>Submit</span></a><ul>
<li><a title="Tracking statuses and editing/creating sra submissions." href="../sra_sub/sub.cgi?view=submissions"><span>Submissions</span></a></li>
<li><a title="Tracking submitted files, their statuses and produced SRA objects." href="../sra_sub/sub.cgi?page=tracking&amp;view=submissions_tracking"><span>Tracking</span></a></li>
<li><a title="Editing preferred name, email, affiliated organizaiton etc." href="../sra_sub/sub.cgi?page=preferences&amp;view=submissions_preferences"><span>Preferences</span></a></li>
<li><a title="Data submission to NCBI (SRA)." href="https://www.ncbi.nlm.nih.gov/sra/docs/submit" target="_blank"><span>Getting started</span></a></li>
<li><a title="Frequently Asked Questions" href="https://www.ncbi.nlm.nih.gov/sra/docs/submitquestions"><span>FAQ</span></a></li>
<!--[if gte IE 5]><iframe></iframe><![endif]-->
</ul>
</li>
<li>
<a title="Software for using and preparing SRA data sets" href="../sra/sra.cgi?view=software"><span>Software</span></a><ul>
<li><a title="Software for using and preparing SRA data sets" href="../sra/sra.cgi?view=software"><span>Download</span></a></li>
<li><a title="SRA Toolkit Documentation" href="../sra/sra.cgi?view=toolkit_doc"><span>Toolkit Documentation</span></a></li>
<li><a title="Sequence SRA XML Schemas" href="../sra/sra.cgi?view=xml_schemas"><span>XML Schema</span></a></li>
<!--[if gte IE 5]><iframe></iframe><![endif]-->
</ul>
</li>
<li class="related first">
<a title="Trace Archive" href="../?view=list_arrivals"><span>Trace Archive
    </span></a><ul>
<li><a title="Trace Archive Main Page" href="../?view=list_arrivals"><span>Main</span></a></li>
<li><a title="How to obtain data" href="../?view=search"><span>Obtaining Data</span></a></li>
<li><a title="Display Archive Statistics" href="../?view=statistics"><span>Statistics</span></a></li>
<li><a title="Tracking submissions" href="../?view=tracking"><span>Tracking</span></a></li>
<li><a title="Trace Archive Documentation" href="../?view=doc_goals"><span>Documentation</span></a></li>
<!--[if gte IE 5]><iframe></iframe><![endif]-->
</ul>
</li>
<li class="related">
<a title="Trace Assembly" href="../assembly?view=taxonomy_tree"><span>
  Trace Assembly
  </span></a><ul>
<li><a title="Trace Assembly Archive Browser" href="../assembly/?view=taxonomy_tree"><span>Main</span></a></li>
<li><a title="Show Submissions" href="../assembly/?view=list_of_submission"><span>Tracking</span></a></li>
<li><a title="Trace Assembly Archive Documentation" href="../assembly/?view=rfc"><span>Documentation</span></a></li>
<!--[if gte IE 5]><iframe></iframe><![endif]-->
</ul>
</li>
<li class="related"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE=Nucleotides&amp;PROGRAM=blastn&amp;BLAST_SPEC=TraceArchive&amp;BLAST_PROGRAMS=megaBlast&amp;PAGE_TYPE=BlastSearch" title="Trace Basic Local Alignment Search Tool"><span>Trace BLAST</span></a></li>
</ul></div>
</div><noscript><div class="warning">Warning: The NCBI web site requires JavaScript for full functionality.<a href="https://www.ncbi.nlm.nih.gov/corehtml/query/static/unsupported-browser.html#enablejs">more...</a>
</div></noscript><div id="gsd"></div><div id="maincontent">
<script type="text/javascript">var oRuntimeData = {
MEMBER: {
"default":"ERS000126"
} 
};</script><div id="sra-viewer-app">
<div id="error-message" class="error" style="display:none">
<div> </div>
<button>Hide message</button>
</div>
<div class="page-title">
<div class="expand closed">
<a href="#" class="expand-handler"><label class="more" for="run-accession">
              Change accession...
            </label><label class="less" for="run-accession">
              Change accession:
            </label></a><span class="expand-body"><form method="get" action="?" style="display:inline;">
               
              <input type="text" name="run" id="run-accession" value="ERR004400" size="10">
               
              <input type="submit" value="Search">
</form></span>
</div>
<h2><span>(ERR004400)</span></h2>
</div>
<ul class="gui-tab" rel="menu1">
<li rel="meta"><a href="#"><span>Metadata</span></a></li>
<li rel="analysis"><a href="#"><span>Analysis (alpha)</span></a></li>
<li rel="reads"><a href="#"><span>Reads</span></a></li>
<li rel="download"><a href="#"><span>Download</span></a></li>
</ul>
<div class="section menu-content menu1 meta" style="display:none">
<div class="ph run"><div>
<table class="zebra run-metatable">
<tr>
<th>Run</th>
<th>Spots</th>
<th>Bases</th>
<th>Size</th>
<th>GC content</th>
<th>Published</th>
<th>Access Type</th>
</tr>
<tr>
<td><div title="Primary Id">ERR004400</div></td>
<td><span id="total_spots">4.1<span class="file-size mb">M</span></span></td>
<td align="right">142.2<span class="file-size mb">M</span>bp</td>
<td align="right">79.2<span class="file-size mb">M</span>
</td>
<td>41.2%</td>
<td>2010-03-15</td>
<td>public</td>
</tr>
</table>
<style>
      .run-metatable, .quality-count-ph, .spot-descriptor table {width:700px;}
      .quality-count-ph div > span  {padding-right:2px;}
    </style>
<div id="id-quality-count-ph" class="quality-count-ph">
<p>Quality graph <a href="#" onclick="return QualCntGraph()">(bigger)</a><a href="#" onclick="return QualCntGraph()" class="big-graph">(smaller)</a></p>
<div>
<span title="0 : 9,204" style="width:20px"><span style="height:0%;width:100%"> </span></span><span title="1 : 352,231" style="width:20px"><span style="height:0.3%;width:100%"> </span></span><span title="3 : 313,465" style="width:20px"><span style="height:0.3%;width:100%"> </span></span><span title="4 : 270,021" style="width:20px"><span style="height:0.3%;width:100%"> </span></span><span title="5 : 350,563" style="width:20px"><span style="height:0.3%;width:100%"> </span></span><span title="6 : 270" style="width:20px"><span style="height:0%;width:100%"> </span></span><span title="7 : 319,778" style="width:20px"><span style="height:0.3%;width:100%"> </span></span><span title="8 : 420,487" style="width:20px"><span style="height:0.4%;width:100%"> </span></span><span title="9 : 391,705" style="width:20px"><span style="height:0.4%;width:100%"> </span></span><span title="10 : 309,393" style="width:20px"><span style="height:0.3%;width:100%"> </span></span><span title="11 : 405,876" style="width:20px"><span style="height:0.4%;width:100%"> </span></span><span title="12 : 571,959" style="width:20px"><span style="height:0.6%;width:100%"> </span></span><span title="13 : 214,192" style="width:20px"><span style="height:0.2%;width:100%"> </span></span><span title="14 : 529,234" style="width:20px"><span style="height:0.5%;width:100%"> </span></span><span title="15 : 727,503" style="width:20px"><span style="height:0.7%;width:100%"> </span></span><span title="16 : 1,060,980" style="width:20px"><span style="height:1%;width:100%"> </span></span><span title="17 : 1,028,388" style="width:20px"><span style="height:1%;width:100%"> </span></span><span title="18 : 1,572,346" style="width:20px"><span style="height:1.5%;width:100%"> </span></span><span title="19 : 705,663" style="width:20px"><span style="height:0.7%;width:100%"> </span></span><span title="20 : 1,956,932" style="width:20px"><span style="height:1.9%;width:100%"> </span></span><span title="21 : 4,889,799" style="width:20px"><span style="height:4.7%;width:100%"> </span></span><span title="22 : 3,885,287" style="width:20px"><span style="height:3.8%;width:100%"> </span></span><span title="23 : 2,600,816" style="width:20px"><span style="height:2.5%;width:100%"> </span></span><span title="24 : 7,220,607" style="width:20px"><span style="height:7%;width:100%"> </span></span><span title="25 : 7,163,752" style="width:20px"><span style="height:7%;width:100%"> </span></span><span title="26 : 1,936,381" style="width:20px"><span style="height:1.9%;width:100%"> </span></span><span title="27 : 102,961,820" style="width:20px"><span style="height:100%;width:100%"> </span></span><span title="28 : 8,890" style="width:20px"><span style="height:0%;width:100%"> </span></span><span title="29 : 16,048" style="width:20px"><span style="height:0%;width:100%"> </span></span><span title="30 : 2,063" style="width:20px"><span style="height:0%;width:100%"> </span></span><span title="32 : 1,682" style="width:20px"><span style="height:0%;width:100%"> </span></span>
</div>
<div class="axis-x" style="width:682px">
<span class="left">0</span><span class="right">32</span>
</div>
<div class="center">Phred quality score</div>
</div>
<div class="spot-descriptor">
<p>This run has 1 read per spot:</p>
<table cellpadding="0" cellspacing="0" width="700"><tbody><tr>
<td class="Application Read" style="min-width:699px;" title="Application Read #0, average length: 35, stdev: 0, 100%">L=35, 100%</td>
<td class="read-spacer"></td>
</tr></tbody></table>
<div class="help-container xinline spot-descriptor-help"><dl class="expand closed">
<dt><a class="expand-handler" href="#"><span>Legend</span></a></dt>
<dd class="expand-body"><table>
<tr>
<td class="Technical Read" width="100"> </td>
<td> </td>
<td>Technical read</td>
</tr>
<tr>
<td class="Application Read" width="300"> </td>
<td> </td>
<td>Application Read</td>
</tr>
<tr>
<td class="Read" width="300">L=4, 100%</td>
<td> </td>
<td>Length is 4, 100% spots contain this read</td>
</tr>
<tr>
<td class="Read" width="300">̅L=165, σ=92.8, 66%</td>
<td> </td>
<td>Average length is 165, standard deviation is 92.8, 66% spots contain this read</td>
</tr>
</table></dd>
</dl></div>
</div>
<div class="expand inline" id="sra-run-attr-list">
<a class="expand-handler" href="#"><span class="more">Show 2 additional attributes</span><span class="less">Hide attributes</span></a><table class="expand-body zebra run-attributes">
<tr>
<th>Name</th>
<th>Value</th>
</tr>
<tr>
<td>ENA-FIRST-PUBLIC</td>
<td><div style="float:left; width:100%; overflow:auto; *height:2.5em;" class="value">2010-02-26 </div></td>
</tr>
<tr>
<td>ENA-LAST-UPDATE</td>
<td><div style="float:left; width:100%; overflow:auto; *height:2.5em;" class="value">2018-11-16 </div></td>
</tr>
</table>
</div>
</div></div>
<div class="ph experiment"><div><table class="zebra">
<tr>
<th>Experiment</th>
<th>Library Name</th>
<th>Platform</th>
<th>Strategy</th>
<th>Source</th>
<th>Selection</th>
<th>Layout</th>
</tr>
<tr>
<td valign="top">
<a id="exp-accession" href="https://www.ncbi.nlm.nih.gov/sra/ERX000491">ERX000491</a><br><br><a href="https://blast.ncbi.nlm.nih.gov/blast/Blast.cgi?PROGRAM=blastn&amp;BLAST_PROGRAMS=megaBlast&amp;PAGE_TYPE=BlastSearch&amp;BLAST_SPEC=SRA&amp;DB_GROUP=Exp&amp;NUM_ORG=1&amp;EQ_MENU=ERX000491" target="_blank" title="Send this experiment to BLAST"><button>to BLAST</button></a>
</td>
<td>YHCDSA</td>
<td>Illumina</td>
<td>WGS</td>
<td>GENOMIC</td>
<td>RANDOM</td>
<td>SINGLE</td>
</tr>
</table></div></div>
<div class="ph sample"><div><table class="zebra">
<tr>
<th>Biosample</th>
<th>Sample Description</th>
<th>Organism</th>
</tr>
<tr>
<td valign="top">
<a href="https://www.ncbi.nlm.nih.gov/biosample/SAMEA913006">SAMEA913006</a>
              (ERS000126)
            </td>
<td valign="top"></td>
<td valign="top"><a href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&amp;id=9606">Homo sapiens</a></td>
</tr>
</table></div></div>
<div class="ph study"><div>
<table class="zebra" style="max-width:40em">
<tr>
<th>Bioproject</th>
<th>SRA Study</th>
<th>Title</th>
</tr>
<tr>
<td valign="top"><a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJEB2005">PRJEB2005</a></td>
<td valign="top"><a href="?study=ERP000053">ERP000053</a></td>
<td>The diploid genome sequence of an Asian individual</td>
</tr>
</table>
<div class="expand closed">
<a href="#" class="expand-handler"><span class="more">Show abstract</span><span class="less">Hide abstract</span></a><div class="expand-body readable abstract"><p>Here we present the first diploid genome sequence of an Asian individual. The genome was sequenced to 36-fold average coverage using massively parallel sequencing technology. We aligned the short reads onto the NCBI human reference genome to 99.97%% coverage, and guided by the reference genome, we used uniquely mapped reads to assemble a high-quality consensus sequence for 92%% of the Asian individual's genome. We identified approximately 3 million single-nucleotide polymorphisms (SNPs) inside this region, of which 13.6%% were not in the dbSNP database. Genotyping analysis showed that SNP identification had high accuracy and consistency, indicating the high sequence quality of this assembly. We also carried out heterozygote phasing and haplotype prediction against HapMap CHB and JPT haplotypes (Chinese and Japanese, respectively), sequence comparison with the two available individual genomes (J. D. Watson and J. C. Venter), and structural variation identification. These variations were considered for their potential biological impact. Our sequence data and analyses demonstrate the potential usefulness of next-generation sequencing technologies for personal genomics.</p></div>
</div>
</div></div>
</div>
<div class="section menu-content menu1 download readable" style="display:none">
<p style="margin-top:1em">
            You need <strong><a href="/Traces/sra/sra.cgi?view=software">SRA Toolkit</a></strong> to operate on SRA
            runs.
          </p>
<p>
            Default toolkit configuration enables it to find and retrieve SRA runs by
            accession. It also downloads (and cache) only the part of data you really need. For example
            quality scores represent a majority of data volume and you may not need them if you
            dump fasta only (versus fastq). Or if you are looking at particular gene you may not
            need reads aligned to other regions or not aligned at all. Same way if you use
            GATK with enabled SRA support you need only SRA run accessions to fire your process.
          </p>
<p><strong><a href="/Traces/sra/sra.cgi?view=toolkit_doc&amp;f=fastq-dump" target="_blank">fastq-dump</a></strong>
            will dump reads in a number of "standard" fastq and fasta formats.
          </p>
<p><strong><a href="/Traces/sra/sra.cgi?view=toolkit_doc&amp;f=vdb-dump" target="_blank">vdb-dump</a></strong>
            is also capable of producing fasta and fastq (beside other formats). It dumps data much
            faster then fastq-dump but ordering of reads may be different and it does not produce split-read
            multi-file output.
          </p>
<p><strong><a href="/Traces/sra/sra.cgi?view=toolkit_doc&amp;f=prefetch">Prefetch</a></strong>
            tool will help you cache all data in advance if you plan to run data analysis in environment where getting
            data from NCBI at run time is unfeasible.
          </p>
<p>
            To select a list of intersing SRA runs in the scope of experiment, sample or study you may use SRA <strong><a href="/Traces/study/">Run Selector</a></strong>
            either directly or from <strong><a href="/sra">Entrez</a></strong> search.
          </p>
<p>
            Read more at <strong><a href="https://www.ncbi.nlm.nih.gov/books/NBK158899/#SRA_download.downloading_sra_data_using">SRA Knowledge Base</a></strong> on how to download SRA data using command line utilities.
          </p>
</div>
<div class="section menu-content menu1 analysis" style="display:none">
<div class="ph analysis"><div class="expand closed">
<dt>
<span style="color:red"><b>Warning</b></span>:<a class="expand-handler" href="#">experimental software</a>
</dt>
<dd class="expand-body">
<p>
                  The sections below show results of analysis run by software which is still in experimental
                  stage. Please use provided results with a boatload of salt and let us know what you think.
                </p>
<i><p>-- SRA team</p></i>
</dd>
</div></div>
<div class="ph taxon_analysis"><div>
<h2>
                  Taxonomy Analysis
                </h2>
<ul>
<li><span>Unidentified reads: <strong>89.99%</strong></span></li>
<li>
<span>Identified reads: <strong>10.01%</strong></span><div id="id-tax_analysis"></div>
<link href="/core/lw/2.0/css/gui.3/tree.3.css" type="text/css" rel="stylesheet">
<script type="text/javascript" src="js/tax_analysis.js"></script><script type="text/javascript">
            (function () {
            var oTaxAnalysisData = [{"n":"tx131567", "p":"0", "d":{"name":"cellular organisms","percent":"10.01"}}, 
{"n":"tx2759", "p":"tx131567", "d":{"name":"Eukaryota","percent":"10"}}, 
{"n":"tx33154", "p":"tx2759", "d":{"name":"Opisthokonta","percent":"9.92"}}, 
{"n":"tx33208", "p":"tx33154", "d":{"name":"Metazoa","percent":"9.92"}}, 
{"n":"tx6072", "p":"tx33208", "d":{"name":"Eumetazoa","percent":"9.92"}}, 
{"n":"tx33213", "p":"tx6072", "d":{"name":"Bilateria","percent":"9.92"}}, 
{"n":"tx33511", "p":"tx33213", "d":{"name":"Deuterostomia","percent":"9.89"}}, 
{"n":"tx7711", "p":"tx33511", "d":{"name":"Chordata","percent":"9.89"}}, 
{"n":"tx89593", "p":"tx7711", "d":{"name":"Craniata","percent":"9.89"}}, 
{"n":"tx7742", "p":"tx89593", "d":{"name":"Vertebrata","percent":"9.89"}}, 
{"n":"tx7776", "p":"tx7742", "d":{"name":"Gnathostomata","percent":"9.89"}}, 
{"n":"tx117570", "p":"tx7776", "d":{"name":"Teleostomi","percent":"9.89"}}, 
{"n":"tx117571", "p":"tx117570", "d":{"name":"Euteleostomi","percent":"9.89"}}, 
{"n":"tx8287", "p":"tx117571", "d":{"name":"Sarcopterygii","percent":"9.88"}}, 
{"n":"tx1338369", "p":"tx8287", "d":{"name":"Dipnotetrapodomorpha","percent":"9.88"}}, 
{"n":"tx32523", "p":"tx1338369", "d":{"name":"Tetrapoda","percent":"9.88"}}, 
{"n":"tx32524", "p":"tx32523", "d":{"name":"Amniota","percent":"9.88"}}, 
{"n":"tx40674", "p":"tx32524", "d":{"name":"Mammalia","percent":"9.79"}}, 
{"n":"tx32525", "p":"tx40674", "d":{"name":"Theria","percent":"9.79"}}, 
{"n":"tx9347", "p":"tx32525", "d":{"name":"Eutheria","percent":"9.78"}}, 
{"n":"tx1437010", "p":"tx9347", "d":{"name":"Boreoeutheria","percent":"9.65"}}, 
{"n":"tx314146", "p":"tx1437010", "d":{"name":"Euarchontoglires","percent":"9.43"}}, 
{"n":"tx9443", "p":"tx314146", "d":{"name":"Primates","percent":"9.2"}}, 
{"n":"tx376913", "p":"tx9443", "d":{"name":"Haplorrhini","percent":"8.84"}}, 
{"n":"tx314293", "p":"tx376913", "d":{"name":"Simiiformes","percent":"8.74"}}, 
{"n":"tx9526", "p":"tx314293", "d":{"name":"Catarrhini","percent":"7.2"}}, 
{"n":"tx314295", "p":"tx9526", "d":{"name":"Hominoidea","percent":"5.3"}}, 
{"n":"tx9604", "p":"tx314295", "d":{"name":"Hominidae","percent":"4.1"}}, 
{"n":"tx207598", "p":"tx9604", "d":{"name":"Homininae","percent":"3.25"}}, 
{"n":"tx9605", "p":"tx207598", "d":{"name":"Homo","percent":"1.1", "kbp":"1567"}}, 
{"n":"tx9606", "p":"tx9605", "d":{"name":"Homo sapiens","percent":"1.1", "kbp":"1563"}}, 
{"n":"tx9596", "p":"tx207598", "d":{"name":"Pan","percent":"0", "kbp":"5"}}, 
{"n":"tx9598", "p":"tx9596", "c":0, "d":{"name":"Pan troglodytes","percent":"0", "kbp":"5"}}, 
{"n":"tx314294", "p":"tx9526", "d":{"name":"Cercopithecoidea","percent":"0.01", "kbp":"9"}}, 
{"n":"tx9527", "p":"tx314294", "d":{"name":"Cercopithecidae","percent":"0.01", "kbp":"9"}}, 
{"n":"tx9528", "p":"tx9527", "d":{"name":"Cercopithecinae","percent":"0", "kbp":"3"}}, 
{"n":"tx9569", "p":"tx9527", "d":{"name":"Colobinae","percent":"0", "kbp":"1"}}, 
{"n":"tx542827", "p":"tx9569", "d":{"name":"Rhinopithecus","percent":"0", "kbp":"1"}}, 
{"n":"tx61622", "p":"tx542827", "c":0, "d":{"name":"Rhinopithecus roxellana","percent":"0", "kbp":"1"}}, 
{"n":"tx2", "p":"tx131567", "d":{"name":"Bacteria","percent":"0", "kbp":"3"}}, 
0];

            utils.addEvent(window, "load", function () {
            StaticTree(document.getElementById("id-tax_analysis"), "tax_analysis_0_", tree_transform(oTaxAnalysisData),
            function (oD, bLast) {
            var label = oD.d.name + ": <strong>" + oD.d.percent + "%</strong>";
            if ( oD.d.kbp != "" && oD.d.percent == "< 0.01" ) {{ label = label + " (" + oD.d.kbp + " Kbp)" ; }}
            return label;
            }); } ); })();
          </script>
</li>
</ul>
<div style="margin:0.3em;">
<h3>Strong signals</h3>
<table class="zebra" style="font-size:0.9em; padding:2pt;">
<tr>
<th>SuperKingdom</th>
<th>Organism</th>
<th>Rank</th>
<th aligh="right" title="fraction of reads">%%</th>
<th title="total size of attributed reads in Kilo-base-pairs">Kbp</th>
<th title="relative strength versus either cutoff value for taxonomy division or genome size"> weighted score </th>
</tr>
<tr>
<td style="padding:2pt;">Eukaryota</td>
<td style="padding:2pt">Catarrhini</td>
<td style="padding:2pt">parvorder</td>
<td style="padding:2pt" align="right">10.0</td>
<td style="padding:2pt" align="right">14,224</td>
<td style="padding:2pt" align="right" title="covered size versus cutoff for that part of taxonomy tree : 10.0  Mbp
                          ">1.4</td>
</tr>
</table>
</div>
<div class="expand closed">
<dt><a class="expand-handler" href="#"><span>How to read results?</span></a></dt>
<dd class="expand-body">
<p>
                        Results show distribution of reads mapping to specific taxonomy nodes as a percentage
                        of total reads within the analyzed run. In cases where a read maps to more than one related
                        taxonomy node, the read is reported as originating from the lowest shared taxonomic node.
                        So when a read maps to two species belonging to the same genus, it is reported as having
                        originated from their common genus. Under typical conditions where a single organism has
                        been sequenced, the expectations are that reads will map to several taxonomy nodes across
                        the organism’s lineage, and that the number of reads mapping to higher level nodes will
                        be more than those that map to terminal nodes.
                      </p>
<p>
                        STAT results are proportional to the size of sequenced genomes. So given a mixed sample
                        containing several organisms at equal copy number, one expects proportionally more reads
                        to originate from the larger genomes. This means that the percentages reported by STAT
                        will reflect genome size and must be considered against the genomic complexity of the
                        sequenced sample.
                      </p>
</dd>
</div>
<div class="expand closed">
<dt><a class="expand-handler" href="#"><span>How taxonomy analysis is done?</span></a></dt>
<dd class="expand-body">
<h3>Overview</h3>
<p>
                        The NCBI SRA Taxonomy Analysis Tool (STAT) calculates the taxonomic distribution of reads
                        from next generation sequencing runs. This analysis maps individual sequencing reads to a
                        taxonomic hierarchy and reports the taxonomic composition of reads within a sequencing run.
                      </p>
<h3>Method</h3>
<p>
                        STAT maps sequencing reads to a taxonomic hierarchy using a two-step strategy based
                        on exact query read matches to precomputed k-mer dictionary databases. In the first
                        pass a small, a “coarse” reference dictionary database is used to identify organisms
                        matching a read set. In the second pass, organism-specific slices from a “fine” reference
                        dictionary database are used to compute distribution of reads between identified taxonomy
                        classes (species and higher order taxonomy nodes). When multiple taxnodes are mapped for
                        single spot we use the lowest non-ambigous mappimg
                      </p>
<p>
                        STAT k-mer dictionaries are built using an iterative <a href="https://en.wikipedia.org/wiki/MinHash" target="_blank">minhash</a>
                        based approach against reference genomic databases. For every fixed segment length of incoming reference
                        nucleotide sequence, k-mer representing this segment selected based on minimum
                        <a href="https://en.wikipedia.org/wiki/Fowler%E2%80%93Noll%E2%80%93Vo_hash_function#FNV-1_hash" target="_blank">fvn1 hash</a> function.
                        Several strategies were used to enhance the specificity and accuracy of STAT results.
                        Low complexity k-mers composed of &gt;50% homo-polymer or dinucleotide repeats (e.g. AAAAAA or ACACACACACA)
                        were filtered from dictionaries, and discrete k-mers belonging to multiple taxonomic references
                        were “merged” at the lowest common taxonomic node shared between references. Finally, the specificity
                        of representative k-mers was determined by searching against the source reference genomic database.
                        When representative k-mers were found in multiple taxonomic references nodes, they were merged at
                        the lowest common taxonomic node as above.
                      </p>
<h3>Genome references</h3>
<p>
                        The NCBI <a href="https://www.ncbi.nlm.nih.gov/refseq/" target="_blank">refseq_genomic</a> database was supplemented
                        with the validated viral genome set (<a href="https://www.ncbi.nlm.nih.gov/genome/viruses/" target="_blank">RefSeq neighbors</a>)
                        and used as the source for k-mer creation in both “coarse” and “fine” sets.
                      </p>
<h3>Taxonomy hierarchy</h3>
<p>
                        Reference sequences were mapped to the taxonomy hierarchy using the NCBI <a href="https://www.ncbi.nlm.nih.gov/taxonomy" target="_blank">taxonomy</a> database. The database contained 48,180 taxonomy nodes in January, 2017.
                      </p>
<h3>Segment sizes and K-mer selection</h3>
<p>
                        K-mer dictionaries were built by computationally slicing reference genomes into sequential segments and selecting 32-mers to represent each segment.
                        The “coarse” k-mer dictionary uses variable segment lengths, proportional to genomes size and ranging from 200-8000 nt. The “fine” k-mer dictionary
                        uses a constant 64 nt segment length for all genomes (for 32-mer index it gives us 32x reduction in space and io at the cost of expectation that
                        we have at least one error-free 64-mer for every spot )
                      </p>
<h3>F.A.Q.</h3>
<ul>
<li>
<span>Can I get the software?</span><p>
                            Yes. at <a href="https://github.com/ncbi/ngs-tools/tree/tax/tools/tax">github</a><pre style="background-color:#eee; border:1px;">
                              git clone https://github.com/ncbi/ngs-tools.git --branch tax
                              cd ./ngs-tools/tools/tax
                              make Makefile

                              and in ./examples folder you can find helper *.sh scripts
                            </pre></p>
</li>
<li>
<span>How can I cite you?</span><p>No publication yet. We intend to post a preprint soon.</p>
</li>
</ul>
</dd>
</div>
</div></div>
</div>
<div class="menu-content menu1 reads" style="display:none"><form method="get" action="?" id="find-spot-form">
<input type="hidden" name="view" value="run_browser"><input type="hidden" name="run" value=""><div id="sra-spot-data" class="section">
<div class="sra-ctrl"><div id="sra-find-spot">
<div>
<label for="id_x6">Filter: <input type="text" name="search" value="" maxlength="61" id="id_x6" autofocus="autofocus"></label> <button type="button" id="find">Find</button> <input type="button" value="Filtered Download" id="btn_download">
</div>
<div class="help-container inline"><dl class="expand closed">
<dt><a class="expand-handler" href="#"><span>What does it do?</span></a></dt>
<dd class="expand-body">
          This button will lead you to the page where you can subset the data
          using "Filter" field and download the results in FASTQ or FASTA format.
        </dd>
</dl></div>
<br clear="all"><div class="help-container inline"><dl class="expand closed">
<dt><a class="expand-handler" href="#"><span>What can the filter be applied to?</span></a></dt>
<dd class="expand-body">
<ul>
<li>
                              Sequence substring: one of the biological reads for a spot should contain the substring
                              <br>
                              Examples:
                              <a href="?view=run_browser&amp;run=SRR000130&amp;search=ATTGGA">ATTGGA</a>,
                              <a href="?view=run_browser&amp;run=SRR000130&amp;search=%5EATTGGA">^ATTGGA</a>,
                              <a href="?view=run_browser&amp;run=SRR000130&amp;search=ATTGGA%24">ATTGGA$</a>,
                              <a href="?view=run_browser&amp;run=SRR000130&amp;search=ATGDNNAT">ATGDNNAT</a>,
                              <a href="?view=run_browser&amp;run=SRR000130&amp;search=ATGGA%26GCGC">ATGGA&amp;GCGC</a><br>
                              The strings are case insensitive, and belong to either 2NA or 4NA alphabets.
                              <br>
                              String length limited to 29 characters in 4NA alphabet
                              (includes IUPAC substitution codes) or 61 characters in 2NA alphabet (ACGT only).
                              Search is case insensitive and strings may be combined with boolean
                              operators &amp; | ! (AND, OR, NOT)
                              <br>
                              See "<a href="SRA_Substring_Search.pdf">SRA nucleotide search expressions</a>" for more details.
                              <br><span class="warning">
                                Maximum size of Run to be search is
                                1.1<span class="file-size gb">G</span></span>
</li>
<li>
                              Name of a spot you are looking for.
                              <br>
                              Example: <a href="?view=run_browser&amp;run=SRR000130&amp;search=EXWA4RL02G9Z6H">EXWA4RL02G9Z6H</a>
</li>
<li>
                              Name of sample pool member, or "all" for all members.
                              <br>
                              Example: <a href="?view=run_browser&amp;run=SRR031989&amp;member=M22_V2">M22_V2</a>
                              will return all spots assigned to the sample pool member M22_V2 for run SRR031989.
                            </li>
<li>
                              Spot Id.
                              <br>
                              Example: <a href="?view=run_browser&amp;run=SRR031989&amp;search=23">23</a>
</li>
</ul>
<p>
                            Please note that the filter searches across read boundaries within each spot. Thus, pattern matches within technical reads and across paired-end data boundaries will also be returned
                          </p>
</dd>
</dl></div>
</div></div>
<h1 id="spot-data-title"> </h1>
<table width="100%" cellpadding="0" cellspacing="0" border="0" id="spot-data-ph">
<tr>
<td id="pager-ph"></td>
<td id="sra-set-views">View: <span><span class="toggle_slave customize-rid rid"><span class="toggle_slave rid"><label for="id-show_bio"><input type="checkbox" rel="update" name="show_bio" id="id-show_bio">biological reads</label><label for="id-show_tec"><input type="checkbox" rel="update" name="show_tec" id="id-show_tec">technical reads</label></span><label for="id-show_qs"><input type="checkbox" rel="update" name="show_qs" id="id-show_qs" class="toggle_master qs">quality scores</label></span><a href="#" class="toggle_master show_rid plink">advanced options</a></span><br><div class="toggle_slave show_rid hid" id="customize">
<a href="#" class="toggle_master show_rid plink" style="float:right;">Hide</a><div id="read-ctrl">
<label for="id-show_read"><input type="checkbox" rel="update" name="show_read" id="id-show_read" class="toggle_master customize-rid">Read</label><div>
<label for="id_x3"><input type="radio" name="space" value="base" rel="update" id="id_x3" checked> base space</label><label for="id_x4"><input type="radio" name="space" value="color" rel="update" id="id_x4"> color space</label>
</div>
<div id="show_separated"><label for="id_x5"><input type="checkbox" name="rid_split" class="toggle_master rid" rel="update" id="id_x5"> show separated reads</label></div>
<div><span class="toggle_slave qs hid"><label for="id-show_qsb"><input type="checkbox" rel="update" name="show_qsb" id="id-show_qsb">quality score with bases</label></span></div>
<div id="clip_info" style="display:none"><label for="id-clip_info"><input type="checkbox" rel="update" name="clip_info" id="id-clip_info">show clipped</label></div>
<div id="show_qs4">
<label for="id-show_qs4"><input type="checkbox" rel="update" name="show_qs4" id="id-show_qs4" class="toggle_master qs4">show 4 channel quality score</label><span style="margin-left:0.2em;" class="toggle_slave qs4 hid"><label for="id-show_qs4b"><input type="checkbox" rel="update" name="show_qs4b" id="id-show_qs4b">with bases</label></span>
</div>
</div>
<div id="graph-ctrl"><label for="id-show_graph"><input type="checkbox" rel="update" name="show_graph" id="id-show_graph">Intensity graph</label></div>
<div id="sig-ctrl"><label for="id-show_sig_table"><input type="checkbox" rel="update" name="show_sig_table" id="id-show_sig_table">Signals table</label></div>
</div>
</td>
</tr>
<tr>
<td id="spot-list" valign="top"></td>
<td valign="top" width="75%">
<div id="spot-data"> </div>
<div id="spot-graph">
<h1>Intensity graph</h1>
<div id="spot-graph-img"></div>
</div>
<div id="spot-sig-table"> </div>
</td>
</tr>
</table>
</div>
</form></div>
</div>
</div><div id="footer">
<div id="footer-links">
<ul class="pipelist2l">
<li><strong><a href="#mail_form" id="mail_form_send">Write to the Help Desk</a></strong></li>
<li><a href="https://www.nih.gov/about/privacy.htm">Privacy Notice</a></li>
<li><a href="https://www.nih.gov/about/disclaim.htm">Disclaimer</a></li>
<li><a href="https://www.nih.gov/about/access.htm">Accessibility</a></li>
</ul>
<ul class="pipelist2l">
<li><a href="https://www.ncbi.nlm.nih.gov/">National Center for Biotechnology Information</a></li>
<li><a href="https://www.nlm.nih.gov/">U.S. National Library of Medicine</a></li>
</ul>
<ul id="banners">
<li><a href="https://www.nih.gov/" id="link_nih">National Institute of Health</a></li>
<li><a href="https://www.hhs.gov/" id="link_dhhs">United States Department of Health and Human Services</a></li>
<li><a href="http://www.firstgov.gov/" data-ignore-http="true" id="link_firstgov">FirstGov.gov: The U.S. Government's Official Web Portal</a></li>
</ul>
</div>
<div id="page-updated"><span id="last-updated">Last update: Thu Aug  2 10:13:37 EDT 2018</span></div>
<form id="mail_form" class="hid" action="/Traces/mailer.cgi?">
<p><em>
                        In the form below please describe the problem that you
                        encountered. We will do our best to fix it as soon as possible.
                    </em></p>
<p><label for="e-mail"><span>E-mail:</span><input name="email" type="text" size="25" value="" id="e-mail"></label></p>
<p><label for="subj"><span>Subject:</span><input name="subject" type="text" size="25" value="" id="subj"></label></p>
<p><label for="mess"><span>Message:</span><textarea name="message" cols="25" rows="15" id="mess"> </textarea></label></p>
<p><button id="mail_form_send_btn" type="button">Send</button><button id="mail_form_close_btn" type="button">Close</button></p>
</form>
</div></body></html>
